frontend/e2e/utils/localhost.simplereport.gov

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
Suggest: a file or directory name that makes it clear that this is an nginx config file (`nginx.conf`, `e2e/utils/nginx/localhost.simplereport.gov`, etc.).

Reply:
Good idea :+1: